tools/ci_build/github/azure-pipelines/templates/stages/mac-ios-packaging-build-stage.yml

```diff
@@ -142,7 +142,7 @@ stages:
         done
 
         # copy over helper script for use in release pipeline
-        cp tools/ci_build/github/apple/upload_pod_archive_and_update_podspec.sh "$(Build.ArtifactStagingDirectory)"
+        cp tools/ci_build/github/apple/package_release_tasks.py "$(Build.ArtifactStagingDirectory)"
```

Comment:
What calls the new script? Is that manually configured in the CI instead of being via yaml?

Reply:
yes, it's called by the CocoaPods package release pipeline.
https://aiinfra.visualstudio.com/Lotus/_release?definitionId=14&view=mine&_a=releases

AFAIK release pipelines don't support YAML definitions.